cli/includes/helpers.php

```diff
@@ -4,21 +4,22 @@
 
 use Exception;
 use Illuminate\Container\Container;
-use Symfony\Component\Process\Process;
-use Symfony\Component\Console\Output\ConsoleOutput;
 use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Output\ConsoleOutput;
 
 /**
- * Define the ~/.config/valet path as a constant.
+ * Define constants
  */
-define('VALET_LOOPBACK', '127.0.0.1');
-define('VALET_HOME_PATH', $_SERVER['HOME'].'/.config/valet');
-define('VALET_SERVER_PATH', realpath(__DIR__ . '/../../server.php'));
-define('VALET_STATIC_PREFIX', '41c270e4-5535-4daa-b23e-c269744c2f45');
+if (! defined('VALET_LOOPBACK')) {
+    define('VALET_LOOPBACK', '127.0.0.1');
+    define('VALET_HOME_PATH', $_SERVER['HOME'] . '/.config/valet');
+    define('VALET_SERVER_PATH', realpath(__DIR__ . '/../../server.php'));
+    define('VALET_STATIC_PREFIX', '41c270e4-5535-4daa-b23e-c269744c2f45');
 
-define('VALET_LEGACY_HOME_PATH', $_SERVER['HOME'].'/.valet');
+    define('VALET_LEGACY_HOME_PATH', $_SERVER['HOME'] . '/.valet');
```

Comment:
As I mentioned in a chat, I'm also going to be moving the codebase to follow Tlint and PSR-12 styles.

Reply:
Well, PSR-12 for sure. Not 100% sure about Tlint; the only one I'm doing for sure is spaces around the concatenation operator.

Reply:
haha, sweet :)